01-gabbar/configs/00-preview/space.yaml

```diff
@@ -3,11 +3,8 @@ kind: Space
 metadata:
   name: gabbar-pr
   labels:
-    team: gabbar
-    kind: pr
-    stakater.com/workload-monitoring: 'true'
```

Comment:
why are we removing this?

Reply:
its required for monitoring from workload prometheus

Reply:
tenant-operator automatically add those

Reply:
make sense